script/validate_json.rb

```diff
@@ -1,53 +1,87 @@
 #!/usr/bin/env ruby
 
 # Validates JSON files in the _data directory
-# Exits 0 on success
-# Exits 1 upon JSON parsing errors
-# Exits 2 if a file's keys are not in alphanumeric order
-# Exits 3 if any sites.json entries are missing the required 'url' key
+# See ErrorCodes for actual return value
+# Exits 'SUCCESS' on success
+# Exits 'PARSE_FAILED' upon JSON parsing errors
+# Exits 'UNSORTED' if a file's keys are not in alphanumeric order
+# Exits 'MISSING_URL' if any sites.json entries are missing the required 'url' key
+# Exits 'MISSING_DIFFICULTY' if any sites.json entries are missing the required 'difficulty' key
+# Exits 'MISSING_DOMAINS' if any sites.json entries are missing the required 'domains' key
+# Exits 'MISSING_NAME' if any sites.json entries are missing the required 'name' key
+# Exits 'UNEXPECTED_DIFFICULTY' if field contains unexpected value on 'difficulty' key
 
 require 'json'
 
+module ErrorCodes
```

Comment:
`ErrorCodes` suggests this will always have an error response. `ExitCodes` would be more accurate.
```suggestion
module ExitCodes
```

Reply:
Agreed